x/validators/msg.go

```diff
@@ -15,6 +17,15 @@ func (*SetValidatorsMsg) Path() string {
 	return pathUpdate
 }
 
+func (m ValidatorUpdate) Validate() error {
+	if len(m.Pubkey.Data) != 32 ||
+		strings.ToLower(m.Pubkey.Type) != "ed25519" {
+		return errors.WithCode(errInvalidPubKey, CodeInvalidPubKey)
+	}
+	// can power be negative?
```

Comment:
No, I don't think it would make sense.

Reply:
Me neither. Added a condition to prevent this